docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -400,6 +400,22 @@ specified in the following form:
     {"key": "header2", "value": "value2"}
   ]
 
+Envoy also supports adding the Original Client IP and Protocol to the request headers that matches a 
```

Comment:
It would be nice if you could rework these docs a bit to make it a bit more like: https://lyft.github.io/envoy/docs/configuration/http_conn_man/access_log.html#format-rules

We will add new rules in the future and that format will make it easier to do so.